botocore/loaders.py

```diff
@@ -205,9 +207,11 @@ class Loader(object):
     # For convenience we automatically add ~/.aws/models to the data path.
     CUSTOMER_DATA_PATH = os.path.join(os.path.expanduser('~'),
                                       '.aws', 'models')
+    BUILTIN_TOOL_EXTRAS = ['sdk', 'botocore']
 
     def __init__(self, extra_search_paths=None, file_loader=None,
-                 cache=None, include_default_search_paths=True):
+                 cache=None, include_default_search_paths=True,
+                 additional_tool_extras=None):
```

Comment:
I don't think we need `additional_tool_extras` for now.  I'd prefer to just have the equivalent of `include_default_search_paths` which can toggle on/off the behavior for the .extra.json files.